extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.Optimizely.js

```diff
@@ -0,0 +1,19 @@
+/*!
```

Comment:
Few things:
- perhaps name of this method should start with a prefix "get"
- is there really a need to create another file for it? why not to put it it in ve.utils?
- please do comply with VE code formatting standards (comment at the to of the file, and brackets around method calls and statements - if)